src/cli.rs

```diff
@@ -100,6 +108,71 @@ pub enum ScreenCommands {
     },
 }
 
+#[derive(Subcommand, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub enum PlaylistCommands {
```

Comment:
playlist changes as well?

Reply:
yes, playlist commands were added in this PR because I had too many changes tied in here